alarm_gateway.tf

```
@@ -1,14 +1,23 @@
 locals {
-  routes = distinct(sort([for path in length(data.gosoline_application_metadata_definition.main) > 0 ? element(data.gosoline_application_metadata_definition.main, 0).metadata.apiserver.routes : [] : path.path]))
+  servers = length(data.gosoline_application_metadata_definition.main) > 0 ? element(data.gosoline_application_metadata_definition.main, 0).metadata.httpservers : []
```

Can you try whether this one works?
```suggestion
  servers = try(data.gosoline_application_metadata_definition.main[0].metadata.httpservers, [])
```